src/containers/LocationPage/LocationPage.jsx

```diff
@@ -71,6 +71,7 @@ const fixedFields = [
 // TODO: Replace once API is implemented, currently reading in the sample incident data
 // eslint-disable-next-line global-require
 const incidentData = require('./sample_data/demo_incidentData.json');
+// const incidentData = undefined;
```

Comment:
This is complicated here, and I think this just means this PR can't be merged until the API changes are in. I think for now just remove line 74. Once Rui-Jie's changes are in, we won't be pulling incident data from a sample JSON file (which is why we needed line 74 in the first place since 73 caused errors when running 'npm test').